hw_classes.py

```diff
@@ -0,0 +1,164 @@
+# Задача-1
+# У вас есть список(list) IP адрессов. Вам необходимо создать
+# класс который будет иметь методы:
+# 1) Получить список IP адресов
+# 2) Получить список IP адресов в развернутом виде
+# (10.11.12.13 -> 13.12.11.10)
+# 3) Получить список IP адресов без первых октетов
+# (10.11.12.13 -> 11.12.13)
+# 4) Получить список последних октетов IP адресов
+# (10.11.12.13 -> 13)
+
+class AddressIP:
+    def __init__(self, list_ip):
+        self.__list_ip = list_ip
+
+    @property
+    def list_ip(self):
+        return self.__list_ip
+
+    def reversed_list_ip(self):
+        ln_ip = []
+        for ip in self.__list_ip:
+            l_ip = ip.split('.')
+            l_ip.reverse()
+            ln_ip.append('.'.join(l_ip))
+        return ln_ip
+
+    def without_octs(self):
```

Comment:
method names should be answered on question "What to do?" E.G `def get_ip_without_last_octs()`